src/main/java/game/jdbc/picturepoker/GamePlay.java

```diff
@@ -0,0 +1,110 @@
+package game.jdbc.picturepoker;
+
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Map;
+
+public class GamePlay {
+
+    //Relavant variables - current game, and the list of players
+    private Game curGame;
+    private Player playerList[] = new Player[4];
+    private static final int PAYOUTS[] = {55, 30, 15, 0};
+
+    private boolean executeTurn(Player player) {
+        // Step 1: Deal cards at random
+        // Step 2: Take bets
+        // Step 3: Change out the cards which were selected for changing
+        // Step 4: Let the player cry at their new cards.
+
+        // return true upon success, false upon failure.
+        return false;
+    }
+
+    private boolean executeLuigi() {
+        //I believe Luigi should be his own class.
+        // The class will be referenced here.
+        Luigi luigi = new Luigi();
```

Comment:
I agree that Luigi probably can just be set as a player instead which will be easier. Also will be changed eventually.